README.md

```diff
@@ -8,10 +8,9 @@ Full instructions for the setup and use of Run Scanner can be found in the [Run
 
 ## Prerequisites
 
-* JDK 11 or later
-* Tomcat 8 or later
-* C++ build environment
-* jsoncpp
```

Comment:
is jsoncpp no longer needed?

Reply:
Still needed, but it's redundant to mention as it's part of "C++ build environment" and included in the runscanner-illumina documentation